internal/restorer/filerestorer_test.go

```diff
@@ -47,101 +43,102 @@ type _TestData struct {
 	idx filePackTraverser
 }
 
-func (i *_TestData) Lookup(blobID restic.ID, _ restic.BlobType) ([]restic.PackedBlob, bool) {
+func (i *TestRepo) Lookup(blobID restic.ID, _ restic.BlobType) ([]restic.PackedBlob, bool) {
 	packs, found := i.blobs[blobID]
 	return packs, found
 }
 
-func (i *_TestData) packName(pack *packInfo) string {
+func (i *TestRepo) packName(pack *packInfo) string {
 	return i.packsIDToName[pack.id]
 }
-func (i *_TestData) packID(name string) restic.ID {
+
+func (i *TestRepo) packID(name string) restic.ID {
 	return i.packsNameToID[name]
 }
 
-func (i *_TestData) pack(queue *packQueue, name string) *packInfo {
+func (i *TestRepo) pack(queue *packQueue, name string) *packInfo {
 	id := i.packsNameToID[name]
 	return queue.packs[id]
 }
 
-func (i *_TestData) fileContent(file *fileInfo) string {
+func (i *TestRepo) fileContent(file *fileInfo) string {
 	return i.filesPathToContent[file.path]
 }
 
-func _newTestData(_files []_File) *_TestData {
-	type _Pack struct {
+func newTestRepo(content []TestFile) *TestRepo {
```

Comment:
nitpicking and very subjective, `content` is rather generic and is already used for `restic.Node` blobs, I think more specific `files` is better here.

Reply:
Agreed. I've left that out because `files` is already used later in the function, but that can be renamed too.